### ISSUE TITLE
:target doesn’t work because preventDefault is called

### ISSUE THREAD

**Issue by chasenlehara** [canjs/can-route-pushstate#75]
Given a template like this:

```html
<a href="#hash-in-the-url">Link</a>
<p id="hash-in-the-url">Target</p>
```

…and some CSS:

```css
p:target {background: yellow;}
```

…and some JS:

```
import route from 'can-route';
import 'can-route-pushstate';
route('/{page}', { page: 'home' });
```

One would expect that clicking on the link would cause the paragraph to light up yellow, when unfortunately `preventDefault()` gets called on the event: https://github.com/canjs/can-route-pushstate/blob/040e3e19548570aadec022051ac923d25659f7e9/can-route-pushstate.js#L187-L198

**Comment by Psykoral** [comment]
Thanks @chasenlehara I'll keep an eye on this issue, I'm relying on this browser feature working correctly ;)

**Comment by chasenlehara** [comment]
Sorry I didn’t get back to you sooner; we just talked about this issue this afternoon and I’m going to try to get a fix out tomorrow.

**Comment by chasenlehara** [comment]
@Psykoral I have a PR open to fix this and I’ll get it merged & released on Tuesday.

**Comment by Psykoral** [comment]
you rule!!!

**Comment by chasenlehara** [comment]
This should be fixed in [3.2.3](https://github.com/canjs/can-route-pushstate/releases/tag/v3.2.3)
